### DISCUSSION TITLE
Logo

### DISCUSSION THREAD

**Discussion by danieldegroot2** [xspanger3770/GlobalQuake#123] [General]
You might be able to add a logo / embed image for the GitHub repository. See
https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/customizing-your-repositorys-social-media-preview

Alternatively it could be added in the readme.

The image used as Discord server icon is also ok for a temporary logo imo.

**Comment by xspanger3770** [comment]
I already have a social media preview image uploaded (the NZ quake), maybe I'll make it the app's icon
